e2e/flags.go

```diff
@@ -104,6 +104,10 @@ var ShareTestEnv = flag.Bool("share-test-env", false,
 var GitProvider = flag.String("git-provider", Local,
 	"The git provider that hosts the Git repositories. Defaults to local.")
 
+// OCIProvider is the provider that hosts the OCI repositories.
+var OCIProvider = flag.String("oci-provider", Local,
+	"The registry provider that hosts the OCI repositories. Defaults to local.")
```

Comment:
Seems like the description should specify the other valid options. Or is this a URL?

Reply:
Local is currently the only valid option. Other options will be added with subsequent PRs (e.g. artifact registry)